examples/basics/scene/mesh_texture.py

```diff
@@ -28,7 +28,7 @@
 
 shading = None if args.shading == 'none' else args.shading
 mesh = Mesh(vertices, faces, shading=shading, color='white')
-mesh.shininess = 1e-3
+mesh.shading_filter.shininess = 1e+1
```

Comment:
Why the change in shininess?

Reply:
Good point. In short, `shininess` breaks backwards compatibility. `shininess_new = 1 / shininess_previous`.

I adapted to the Phong shading implementation to follow the [book formulas](https://en.wikipedia.org/wiki/Phong_reflection_model#Description) more closely. The `shininess` is now the Phong specular exponent `alpha`. Before, `alpha=1/shininess`. This way, I found that no hard-coded factor is needed anymore, [as before](https://github.com/vispy/vispy/blob/ad1f3ffbde688499d4922597c3b639beba6eea70/vispy/visuals/mesh.py#L92). Subjectively, the shading effect now seems better behaved to me.
So in the proposed changes, `shininess` takes values in `[0, infinity]`, with higher values being more mirror-like surfaces (smaller specular highlight).